content/F2015/F15-event-teaparty.md

```diff
@@ -0,0 +1,21 @@
+title: Big CSters Board Game and Tea Party Social
+Date: 2015-09-23
```

Comment:
Hmmm, I disagree with this---I think the career panel post's date should probably be changed, since the posting date is inaccurate (rather than making them *both* inaccurate as a hack!)